builder/dockerfile/dispatchers.go

```diff
@@ -414,6 +414,8 @@ func cmd(b *Builder, args []string, attributes map[string]bool, original string)
 	}
 
 	b.runConfig.Cmd = strslice.StrSlice(cmdSlice)
+	// set config as already being escaped, this prevents double escaping on windows
+	b.runConfig.ArgsEscaped = true
```

Comment:
I don't fully grok this PR, but if you'd adding this for CMD, do you need to add it for ENTRYPOINT?